.github/workflows/docs.yml

```diff
@@ -1,23 +1,44 @@
 name: Build and Deploy Documents
 
-defaults:
-  run:
-    working-directory: ldk/node
 
 on:
   push:
     branches:
       - main
+      - SIDE-1196
 
 jobs:
-  build:
+  deploy-developer-hub:
+    defaults:
+      run:
+        working-directory: docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - run: npm ci
+      - run: npm run build:production
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I created a deploy key for the olive-helps docs repo and added it's private key to the loop-development-kit repo as a secret named "HELPS_DOCS_DEPLOY_KEY".

Use deploy_key instead of github token by changing this line to:
`deploy_key: ${{ secrets.HELPS_DOCS_DEPLOY_KEY }}`